### DISCUSSION TITLE
use a variable as a part of title for grafana dashboard

### DISCUSSION THREAD

**Discussion by gredwhite** [grafana/grafana#56481] [General]
I have a grafana dashboard for postgres:

```
  ....
  },
  "timezone": "",
  "title": "My PostgreSQL Database dashboard",
  "uid": "postrgresql-database-dashboard",
  "version": 1,
  "weekStart": ""
}
```
I want to make title dynamically generated.

smth like

`"title": ${my_variable} + " My PostgreSQL Database dashboard",`

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
